server/routes/students.js

```diff
@@ -151,4 +175,37 @@ router.patch('/interests/add/:id', async (req, res, next) => {
     }
 })
 
+router.patch('/college/update/:id', passport.authenticate('jwt', {session: false}), async (req, res, next) => {
```

Comment:
If we do allow a single college addition, we can just send the id of the college and push into the `collegeShortlist` field in a student record! We wouldn't need to include the logic to check if the college was a new one vs. an existing one!